vshn/website
# feat(components): add typing animation in Hero for Home page

**Describe what changes this pull request brings**
- This pull request adds typing animation for the title in Hero for Home page.


**Describe any additional changes**
N/A


**Steps to test**

1. Open [the page](https://typing-animation--vshn.netlify.app/PAGE_URL)
2. Confirm that everything looks and works as expected


**Screenshots**

https://user-images.githubusercontent.com/48465000/102908721-843c9e80-4480-11eb-96bb-f270008e7751.mov




**Checklist**



**TODO**
There was found a bug in the implementation of the existing component Typewriter. Since I came up with the idea of using typewritter-effect instead.  


**References**
- Close the task in Notion [Add typing animation for application word on the home page](https://www.notion.so/pixelpoint/Add-typing-animation-for-application-word-on-the-home-page-4bbe12b95e3845eb80681710de57b319)
